src/args.rs

```diff
@@ -60,46 +60,46 @@ pub struct Arguments {
     ///
     /// This is an alias for `--format=terse`. If this is set, `format` is
     /// `None`.
-    #[clap(
+    #[arg(
         short = 'q',
-        long = "--quiet",
+        long = "quiet",
         conflicts_with = "format",
         help = "Display one character per test instead of one line. Alias to --format=terse",
     )]
     pub quiet: bool,
 
     // ============== OPTIONS =================================================
     /// Number of threads used for parallel testing.
-    #[clap(
-        long = "--test-threads",
+    #[arg(
+        long = "test-threads",
         help = "Number of threads used for running tests in parallel. If set to 1, \n\
             all tests are run in the main thread.",
     )]
     pub test_threads: Option<usize>,
 
     /// Path of the logfile. If specified, everything will be written into the
     /// file instead of stdout.
-    #[clap(
-        long = "--logfile",
+    #[arg(
+        long = "logfile",
         value_name = "PATH",
         help = "Write logs to the specified file instead of stdout",
     )]
     pub logfile: Option<String>,
 
     /// A list of filters. Tests whose names contain parts of any of these
     /// filters are skipped.
-    #[clap(
-        long = "--skip",
+    #[arg(
+        long = "skip",
         value_name = "FILTER",
-        number_of_values = 1,
+        num_args = 1,
         help = "Skip tests whose names contain FILTER (this flag can be used multiple times)",
     )]
     pub skip: Vec<String>,
```

Comment:
FYI `num_args = 1` is likely not needed.  Most likely `number_of_values = 1` was being set back from the structopt days when a `Vec` implied `multiple = true`.  clap v3 split `multiple` into `multiple_values` and `multiple_occurrences` and `Vec` only implies `multiple_occurrences`.

Reply:
Thanks!